application/actions/app.go

```diff
@@ -66,6 +66,8 @@ func App() *buffalo.App {
 
 		app.POST("/gql/", gqlHandler)
 
+		app.GET("/conversations/", usersThreads)
```

Comment:
do we need trailing slash here? Does that then require it on the API?

Reply:
:+1:  (I just did like the others)